dhcpv4/bindtointerface_linux.go

```diff
@@ -0,0 +1,47 @@
+// +build linux
```

Comment:
I think this can be dropped in favor of `NewIPv4UDPConn` https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/nclient4/conn_linux.go#L33